### PR TITLE
upgraded to version 1.4 api

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Upgraded to latest api version 
Removed the derived query decorated here

### CODE REVIEW

File: nft-demo/src/App.js [hunk 1]
```diff
@@ -105,6 +106,7 @@ async function extractMeta(api) {
   if (specTypes.ExtrinsicPayloadV4) {
     delete specTypes.ExtrinsicPayloadV4;
   }
+  delete specTypes.EnhancedTokenId;
```
Comment (jordy25519) [inline nft-demo/src/App.js]: what does this do?

**Review by jordy25519 — APPROVED** [review]
lgtm

File: nft-demo/src/App.js (hunk 1 above)
Comment (KarishmaBothara) [reply nft-demo/src/App.js]: Need to remove all the classes/objects from the spectypes.
The reason for the removal of the following object from specTypes -
the metadata that gets updated on the CENNZnet extension requires types of type (Record<string, string>)
It does not allow classes/objects...

I am planning to have a different way of filtering this

**Review by eupston — APPROVED** [review]
lgtm nice one!

**Comment by KarishmaBothara** [comment]
Closing this as rebase was difficult due to restrict..
addressed in https://github.com/cennznet/NFTDemo/pull/5

**Pull request closed without merging**